src/utils/autoManagerAssigner.js

```diff
@@ -0,0 +1,4 @@
+const assignManger = (managersList) => {
+    return managersList[Math.floor(Math.random() * managersList.length)];
+}
+export default assignManger;
```

Comment:
@hakoemmy I suggest you change the spelling, use assignManager instead of assignManger

Reply:
@AbdoulNuru Sure thing, thanks for FRESH EYES. let me take it into consideration